### ISSUE TITLE
["BUG"] x smart --app is not displaying correctly | [mod=smart] [v0.4.12]

### ISSUE THREAD

**Issue by Oveln** [x-cmd/x-cmd#86]
**Describe the bug**

**System (please complete the following information):**
 - Version: [A sha512, type 'x version' so you can get the version sum]
 - Shell: [Zsh]
 - OS: [Linux Fedora]
 - Arch: [amd64]

**Screenshots**
```
x smart --app
X-CMD module smart is a shallow wrapper of package smartctl.
We provide a simple guiding tui. Ctrl-C to interupt.
If you want to know more, use `x smart -h`
——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
/home/oveln/.x-cmd.root/v/.914fcf6e/mod/pick/lib/main: line 102: ___x_cmd_smart___list_disk: command not found
- E|pick: |
    The data is empty
- I|smart: require priviledge using sudo module
- I|x: showing SMART for disk ==>
  more:
    command: x sudo x smartctl -a ""
——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
smartctl 7.2 (build date Dec  7 2021) [x86_64-linux-6.10.10-200.fc40.x86_64] (local build)
Copyright (C) 2002-20, Bruce Allen, Christian Franke, www.smartmontools.org

: Unable to detect device type
Please specify device type with the -d option.

Use smartctl -h to get a usage summary

```

**To Reproduce**
use ```x smart --app```

**Comment by curry798** [comment]
Thank you for your feedback. 
We will fix this in version 0.4.12.

**Comment by edwinjhlee** [comment]
Already fixed and will be released on next Wednesday. ( 20240925 )

BTW, `x smart` == `x smart --app`

**Comment by edwinjhlee** [comment]
This issue should be resolved in v0.4.12 ~
Please reopen this issue if the problem persists.

Thanks for your support ~
